Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/impl/apmlog/LogStatsControllerWithAPMLog.java

```diff
@@ -95,6 +125,73 @@ public void initializeControls(Object data) {
         AttributeLogSummary oriLogSummary = parent.getLogData().getAttributeLog().getOriginalLogSummary();
         updateFromLogSummary(oriLogSummary, oriLogSummary);
     }
+    
+    @Override
+    public void initializeEventListeners(Object data) throws Exception {
```

Comment:
The event handler registrations in this method could be abbreviated as lambda expressions.

Reply:
Again, please check the readability of the code rather than always jumping to lambda. I see lambda makes the code look nice in some cases, e.g. only one line of code, but not always. Please check if there is any readability issue here.

Reply:
It is always going to be easier to read without the uninformative inner class constructors.  They add nothing to make the code easier to understand.  You know you're going to pass an EventListener the moment you write addEventListener, so saying it twice just reduces the density of meaningful code and adds two levels of indentation.